docs/develop/api-reference/index.mdx

```diff
@@ -196,6 +196,96 @@ puts client.caches.to_a.join(", ")
   cli={`momento cache list`}
 />
 
+### Flush cache
+
+Flushes all data from a cache
+
+Attributes:
+
+| Name      | Type   | Description                      |
+| --------- | ------ | -------------------------------- |
+| cacheName | String | Name of the cache to be flushed. |
+
+:::tip
+
+While you could use [Delete Cache](#delete-cache), then [Create Cache](#create-cache) to mimic this, the FlushCache API keeps the settings and only deletes the data in the cache.
+
+:::
+
+<SdkExamples
+  js={`
+const { SimpleCacheClient, EnvMomentoTokenProvider, LogLevel, LogFormat, Configurations } = require('@gomomento/sdk');
+const credentials = new StringMomentoTokenProvider ({ authToken: process.env.MOMENTO_AUTH_TOKEN});\n
+const momento = new SimpleCacheClient({
+  configuration: Configurations.Laptop.latest(),
+  credentialProvider: credentials,
+  defaultTtlSeconds: 60
+});
+`}
+  python={`
+import momento.simple_cache_client as scc\n
+_MOMENTO_AUTH_TOKEN = "eyJhbGc.MyTestToken"
+_ITEM_DEFAULT_TTL_SECONDS = 15\n
+with scc.SimpleCacheClient(_MOMENTO_AUTH_TOKEN, _ITEM_DEFAULT_TTL_SECONDS) as cache_client:
+    cache_client.flush_cache('test-cache')
+`}
+  java={`
+String MOMENTO_AUTH_TOKEN = "eyJhbGc.MyTestToken";
+int DEFAULT_ITEM_TTL_SECONDS = 15;\n
+SimpleCacheClient simpleCacheClient = SimpleCacheClient
+    .builder(MOMENTO_AUTH_TOKEN, DEFAULT_ITEM_TTL_SECONDS)
+    .build()\n
+simpleCacheClient.flushCache("test-cache");
+    `}
+  go={`
+const (
+    authToken             = "eyJhbGc.MyTestToken"
+    itemDefaultTtlSeconds = 15
+)
+client, err := momento.NewSimpleCacheClient(authToken, itemDefaultTtlSeconds)
+if err != nil {
+    panic(err)
+}
+err = client.FlushCache(&momento.FlushCacheRequest{
+    CacheName: "test-cache",
+})
+    `}
+  csharp={`
+TimeSpan DEFAULT_TTL = TimeSpan.FromSeconds(15);
+ICredentialProvider authProvider = new StringMomentoTokenProvider("eyJhbGc.MyTestToken");
+using SimpleCacheClient client = new SimpleCacheClient(Configurations.Laptop.Latest(), authProvider, DEFAULT_TTL);
+await client.FlushCacheAsync("test-cache");
+    `}
+  rust={`
+let auth_token = "eyJhbGc.MyTestToken";
+let item_default_ttl_seconds = 15;
+let mut cache_client = SimpleCacheClientBuilder::new(
+    auth_token,
+    NonZeroU64::new(item_default_ttl_seconds).unwrap(),
+)
+.unwrap()
+.build();\n
+let cache_name = String::from("test-cache");
+match cache_client.flush_cache(&cache_name).await {
+    Ok(_) => {}
+    Err(err) => {
+        eprintln!("{}", err);
+    }
+}
+`}
+  ruby={`
+require 'momento'
+MOMENTO_AUTH_TOKEN = "eyJhbGc.MyTestToken"
+DEFAULT_TTL_SECONDS = 15
+client = Momento::SimpleCacheClient.new(
+  auth_token: MOMENTO_AUTH_TOKEN, default_ttl: DEFAULT_TTL_SECONDS
+)
+response = client.flush_cache('test-cache')
+raise repsonse.error if response.error?
+  `}
+  cli={`momento cache flush-cache --name test-cache`}
```

Comment:
AFAIK this doesn't exist yet, do we still want to document it?  I presume that is true for some of the stuff above like ruby too?

Reply:
Correct it doesn't exist yet for all of these. I can remove it for now if you want me to.

Reply:
Removed the code examples for now.